### DISCUSSION TITLE
Linux (specifically Ubuntu) installation process seems completely broken

### DISCUSSION THREAD

**Discussion by cardonator** [microsoft/openjdk#74] [Q&A] [answered]
I was trying to install Microsoft's OpenJDK 11 flavor on Ubuntu and ran into many roadblocks using Ubuntu 20.04 as the base OS candidate.

The command in the instructions:
```
wget https://packages.microsoft.com/config/ubuntu/{version}/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
```

Does not work for any version of Ubuntu. It's always returning a 404. I got around this by manually grabbing the list from the same directory and adding it to my sources, and then downloading the key from packages.microsoft.com and manually adding it to the apt key ring. However, then I couldn't install the package because although it was listed in the Apt repo, the file is also missing from packages.microsoft.com for Ubuntu.

Am I looking in the wrong place? Is the build system just broken right now?

**Comment by brstaffoMS** [comment, marked as answer]
Hi @cardonator. There is a known outage that appears to have caused this issue for you and others. Work is underway to get services working normally again. I don't know what the timeline is for restoration, but please try again tomorrow; hopefully it will be resolved by then.

**Reply by karianna** [reply]
FYI - 09:54 UK time - this is still an issue, we'll try to find out what's going on upstream

**Reply by cardonator** [reply]
Thanks for the update!

**Reply by karianna** [reply]
This has now been resolved.
